scripts/deploy-aws.sh

```diff
@@ -46,30 +48,41 @@ juju add-relation aws-integrator kubernetes-master
 juju add-relation aws-integrator kubernetes-worker
 
 # Wait for cloud to finish booting up
-juju wait -e $CONTROLLER:$AWS_MODEL -w
+juju wait -e $CONTROLLER:$AWS_MODEL -vw
 
 juju expose kubeapi-load-balancer
 
 # Copy details of cloud locally, and tell juju about it
 juju scp -m $AWS_MODEL kubernetes-master/0:~/config $KUBECONFIG
 
-juju add-k8s $CLOUD -c $CONTROLLER
+juju add-k8s $CLOUD -c $CONTROLLER --region=ec2/$REGION --storage juju-operator-storage
 juju add-model $MODEL $CLOUD
 
 # Set up some storage for the new cloud, deploy Kubeflow, and wait for
 # Kubeflow to boot up
 kubectl --kubeconfig=$KUBECONFIG create -f storage/aws-ebs.yml
-juju create-storage-pool operator-storage kubernetes storage-class=juju-operator-storage storage-provisioner=kubernetes.io/aws-ebs parameters.type=gp2
 juju create-storage-pool k8s-ebs kubernetes storage-class=juju-ebs storage-provisioner=kubernetes.io/aws-ebs parameters.type=gp2
-juju deploy kubeflow --channel $CHANNEL
-juju wait -e $CONTROLLER:$MODEL -w
+
+# Allow building local bundle.yaml, otherwise deploy from the charm store
+if [[ "$BUILD" = true ]] ; then
+    juju bundle deploy --build
```

Comment:
```bash
$ juju version
2.6-rc1-xenial-amd64

$ juju bundle
ERROR unrecognized command: juju bundle
```

Was `juju bundle` a typo, or is that available in some newer juju?

Reply:
That's a snap package that I've published that adds the `juju-bundle` plugin. It allows you to build a bundle's charms and deploy them, but if you just want to test the script out, you can just not pass in `BUILD=true`, and it will use the charm store charms.

Reply:
> That's a snap package that I've published that adds the `juju-bundle` plugin

Maybe add `juju-bundle` to the `missing` check at the top (at least when `BUILD=true`)?